src/charm.py

```diff
@@ -17,6 +17,7 @@
 from state import State
 
 logger = logging.getLogger(__name__)
+WORKLOAD_VERSION = "1.21.5"
```

Comment:
is there a way to get this in a more dynamic way?

Reply:
I have not found a way yet to programatically query the version yet. I can fetch it from `metadata.yaml`, but this will not work for `temporal-k8s` as we have are not using the upstream image there. 

Either way, we will need to update this value when we want to update the version of the upstream image we are using so that the charm releases a new version.

Reply:
i think that would be cool... i saw some other charms getting it from the snap version, or others getting it from some library they import. For the temporal, i guess the version of the rock should include the version of upstream temporal